File: jpa/src/main/java/com/jpa/configuration/exceptionHandler/exception/ErrorResponse.java
package com.jpa.configuration.exceptionHandler.exception;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ErrorResponse implements Serializable {
    private String timestamp;
    private Integer status;
    private String message;
    private String path;
}

File: security/src/main/kotlin/com/security/config/exceptionHandler/exceptions/ErrorResponse.kt
package com.security.config.exceptionHandler.exceptions

import java.io.Serializable

class ErrorResponse(
    val timestamp: String? = null,
    val status: Int? = null,
    val message: String? = null,
    val path: String? = null,
    val trace: String? = null
) : Serializable